lblod/app-digitaal-loket
# Feature: new lokaal bestuurlijk talent subsidy

## ID
DGS-184

## Description

New subsidy 'Beurs Lokaal Bestuurlijk Talent'. The subsidy should be available for:

- gemeente
  -> Gemeente Aalst 
- OCMW
  -> OCMW Aalter
- provinciebestuur
   -> Provincie Antwerpen
- intergemeentelijk samenwerkingsverband
  -> Intercommunale Leiedal 
- welzijnsvereniging
  -> OCMW vereniging Welzijnsvereniging Sint-Gillis-Waas 
- Autonoom gemeentebedrijf
  -> Autonoom gemeentebedrijf AGB Cultuur Geel 
- Autonoom provinciebedrijf
  -> Autonoom provinciebedrijf APB Gouverneur Kinsbergen Centrum 

Make sure you use the latest version of ember-submission-form-fields when reviewing, see https://github.com/lblod/frontend-loket/pull/376 or use the frontend docker container

## Type of change


## How to test

Log in as the different organisations and verify working subsidy 'Beurs Lokaal Bestuurlijk Talent'.

## Links to other PR's

- https://github.com/lblod/frontend-loket/pull/377
